### PR TITLE
bugfix: Navigate method didn't work with Windows-style paths

### LINKED ISSUE #150: config file doesn't work
version:0.3.4
platform:win10 x64

when i set root path in the file arduino-cli.yaml, it doesn't work:
Windows:
   - paths:
     - sketchbook_path: ./
     - arduino_data: ./
   - proxy:
     - type: auto

the cli tool download path is still in C:\Users\***\AppData\Local

### PR DESCRIPTION
**Pull request by cmaglie**
Fix #150 

/cc @berry0511 @SebCanet

### CODE REVIEW

File: configs/navigate.go [hunk 1]
```diff
@@ -18,24 +18,22 @@
...
 )
 
-func (c *Configuration) Navigate(root, pwd string) {
-	relativePath, err := filepath.Rel(root, pwd)
-	if err != nil {
-		return
-	}
+func (c *Configuration) Navigate(pwd *paths.Path) {
+	parents := pwd.Clean().Parents()
+	fmt.Println(parents)
```
Comment (matteosuppo) [inline configs/navigate.go]: Maybe let's get rid of this debug line

Comment (cmaglie) [reply configs/navigate.go]: uhm... good point

**Comment by SebCanet** [comment]
When click on download: 404 Not Found

**Comment by cmaglie** [comment]
@SebCanet I've pushed another fix and ArduinoBot produced another build, please test this one:

http://downloads.arduino.cc/PR/arduino-cli/arduino-cli-45-PR154-windows.zip

**Comment by SebCanet** [comment]
Everything wokrs! Great, thanks.

**Pull request merged**